templates/ovncontroller/bin/liveness.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+# Ensure ovn-controller pid is up
+output=$(pidof ovn-controller)
+if [ "$output" != "1" ]; then
+    echo "ERROR - OVN-Controller is not running"
+    exit 1
+fi
+
+# Ensure ovn-controller is running and not paused, this running status is not
+# affected by any other services/depedencies as the connection-status output
+output=$(ovn-appctl -t ovn-controller debug/status)
+if [ "$output" != "running" ]; then
```

Comment:
After taking a loop to that test file, looks like the prefix x is commonly used in shell scripts to avoid issues with empty strings. If the command output is an empty string, then x$(...) will simply be x, which prevents comparison errors when there is no concrete output, so let me do that kind of comparison on the code.

Reply:
Finally checking debug/status command is getting normal output and no issues where there running the command, if it pass then checking "running" output